perl-file-rsyncp/riscv64.patch

```diff
@@ -0,0 +1,12 @@
+diff --git PKGBUILD PKGBUILD
+index a5c6bcc..5292eca 100644
+--- PKGBUILD
++++ PKGBUILD
+@@ -24,6 +24,7 @@ prepare() {
+ build() {
+   cd File-RsyncP-$pkgver
+   USE_DEFAULT=1 perl Makefile.PL INSTALLDIRS=vendor
++  cp -vf /usr/share/autoconf/build-aux/config.guess FileList/
```

Comment:
If the file wasn't generated by the above Makefile.PL call (it's unlikely, but if so, please fix the file used to generate it), this should be moved to prepare().

Reply:
Fixed it.
I'll notice it in the future, thanks.